sdk/core/Azure.Core/perf/HttpPipelineBenchmark.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+
+namespace Azure.Core.Perf;
+
+[SimpleJob(RuntimeMoniker.Net60)]
```

Comment:
Could you add an allocation anayzer to show how much each test allocates?

Reply:
I had originally added this, but then saw that it was added automatically in the [benchmark runner's config](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/perf/Program.cs#L33).  I'm fine adding it explicitly to the class definition if it has no effect and someone wanted to run it outside this program file implementation.

Reply:
Ah, I did not realize we have that in the config.

Reply:
I am serious that I literally typed it in and then went and checked and deleted it.  I am happy to add it back so we don't make this mistake in the future - we have both spent more time on it now than we needed to :)

Let me see if adding it as an attribute too has any negative side-effect.

Reply:
Looks like there's no problem with this; will add